python/cuml/cuml/cluster/dbscan.pyx

```diff
@@ -344,6 +345,10 @@ class DBSCAN(Base,
             check_dtype=[np.float32, np.float64]
         )
 
+        if n_rows * n_cols > (2**31 - 1):
+            out_dtype = "int64"
+            logger.warn("Using int64 for out_dtype because n_rows * n_cols >= INT_MAX")
```

Comment:
does this need to be a warning? Can't it just be `info`? 
```suggestion
            logger.info("Using int64 for out_dtype because n_rows * n_cols >= INT_MAX")
```